docs/source/blueprints/BP000.rst

```diff
@@ -273,11 +276,15 @@ section:
    of Avocado? Should we deprecate some modules, classes, or methods? Are we
    going to keep backwards compatibility or not?
 
- * Security Implications: Do you have any concerns about security with your
-   proposed solution and what are they?
+ * Security Implications: Do you have any concerns about security with
```

Comment:
Usually, we don't change approved blueprints. But I will close my eyes on this since are very small and we are still learning.

Reply:
We can keep this text only at the template if you prefer.

Reply:
> We can keep this text only at the template if you prefer.

Or, we can think of BP000 as a "not" BluePrint, because it's really a meta BluePrint, right?

Reply:
IMO, *it is* a blueprint. But a meta one.